index.html

```diff
@@ -30,26 +30,29 @@ <h1>I'm Redwan.<br/>
                 </p>
             </div>
             <!-- End of header top -->
-
             <div class="headerBottom">
                 <a href="#">LET’S CONNECT</a>
                 <div class="link-btn">
-                    <img src="images/Twitter.svg" alt="Twitter"/>
-                    <span>
-                        <img src="images/Instagram.png" alt="Instagram"/>
-                        <img src="Images/VectorN.png" alt="Instagram"/>
-                    </span>
-                    
-                    <img src="images/link-Vector3.svg" alt=""/>
-                    <img src="images/link-Vector4.svg" alt=""/>
-                    <img src="images/link-Vector5.svg" alt=""/>
+                    <ul>
+                        <li>
+                            <a href="#"><img src="images/Twitter.svg" alt="Twitter"/></a>
+                        </li>
+                        <li>
+                            <a href="#"><img src="images/Instagram.png" alt="Instagram"/><img src="Images/VectorN.png" alt="Instagram"/></a>
```

Comment:
- [ ] As suggested by previous reviewer, kindly fix your image path. You are using a capital `I` instead of a small letter `i`. Kindly fix this. RIght URL `images/VectorN.png`.